internal/deploy/manual.go

```diff
@@ -0,0 +1,92 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package deploy
+
+import (
+	"context"
+
+	"github.com/elastic/e2e-testing/internal/shell"
+	"go.elastic.co/apm"
+)
+
+// manualDeploymentManifest deploy manifest for docker
+type manualDeploymentManifest struct {
```

Comment:
I see the `manualDeployment` abstraction a bit complex to understand. I.e. the span label `manual.manifest.execIn`

Have you thought other options instead of manual?

Reply:
So I thought about some other names:

- mixed
- heterogeneous
- remote

I didnt particularly care for any of them (including manual) so any suggestions on naming is welcome!

Reply:
I was thinking about `local`, but after reading your comments, it's clear to me that it's not enough, as you expressed the concerns properly: it's not local because we install the agent locally connecting to a remote stack. Maybe `remote` makes more sense